src/app/games/mancala/awale/AwaleTutorial.ts

```diff
@@ -1,28 +1,34 @@
-import { AwaleMove } from 'src/app/games/awale/AwaleMove';
-import { AwaleState } from 'src/app/games/awale/AwaleState';
+import { AwaleMove } from 'src/app/games/mancala/awale/AwaleMove';
+import { MancalaState } from 'src/app/games/mancala/commons/MancalaState';
 import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
 
 export class AwaleTutorial extends Tutorial {
 
     public tutorial: TutorialStep[] = [
-        TutorialStep.anyMove(
+        TutorialStep.informational(
+            $localize`Mancala`,
+            $localize`Mancala is the name of a family of board games that date back at least to the third century. Mancalas are games of distribution (sowing) and capture. Their goal is to capture the most seeds. The spaces in Mancalas are called the houses. As you're playing Dark, the 6 houses on the bottom are yours.`,
+            MancalaState.getInitialState(),
+        ),
+        TutorialStep.informational(
+            $localize`Awalé`,
+            $localize`Bonus fact: Awalé is the more common of all Mancalas.`,
```

Comment:
peut-être merger ça dans l'informationnel précédent ? Genre : Awalé is a Mancala. Mancala is the name of ...

Car sinon si je lis le tuto du jeu ABC et que la première étape du tuto parle de XYZ sans mentionner ABC, c'est un peu "wtf, j'me suis trompé de page ou y'a un bug?"

Reply:
up!

Reply:
Certes mais ça permet d'avoir une intro en communs à ces jeux d'une même famille, qu'on ne traduit qu'une fois, faut bien une intro non ?

Reply:
$localize`${this.gameName} is a Mancala. Mancalas are ...` ou du genre ?